### PR TITLE
Feat/update release notes page

### PR DESCRIPTION
**Pull request by lorenzolewis**
Re-themes the Release Notes page and also moves it from a "page" to a "doc"
- Fix async issue that caused page to lag when loading release notes
- Fall back to using Docusaurus Tab component
- Uses `react-markdown` to render the raw markdown data into HTML
- Remove now un-needed `debounce` utility
- Add link to GitHub for each link (can act as a fallback if needed in the future)

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -13,3 +13,7 @@ to = "https://docs.rs/tauri/1.0.0-rc.0/tauri/"
 [[redirects]]
 from = "/partners"
 to = "/#sponsors"
+
+[[redirects]]
+from = "/release-notes"
+to = "/docs/release-notes"
```
Comment (FabianLars) [inline netlify.toml]: That's not correct (anymore)

File: package.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
     "react-apexcharts": "1.4.0",
     "react-content-loader": "6.2.0",
     "react-dom": "17.0.2",
-    "react-markdown": "8.0.3",
+    "react-markdown": "^8.0.3",
```
Comment (FabianLars) [inline package.json]: Doesn't matter too much, but renovate will revert that one (same for the lock file)

**Review by FabianLars — CHANGES_REQUESTED** [review]
all in all pretty good imo.
My only problem are the dropdowns. Are they generally modifiable? They don't look clickable & title-ish enough imo.
If we can't really modify them, can we at least change the cursor on hover?

**Comment by lorenzolewis** [comment]
@FabianLars how about now? Set the titles to h3 and added a bit of CSS to make it more "link-like"

**Comment by lorenzolewis** [comment]
Also, does your browser at all lag when data is loading on this page? There was an issue where it kept having performance issues but I updated Docusaurus and I'm not running into the issue now, so just want to confirm.

**Comment by FabianLars** [comment]
>how about now?

I like it :+1: 

> performance

Wayyy better than on the live site :+1:

**Comment by JonasKruckenberg** [comment]
@lorenzolewis did you decide that the search (filter) wasn't worth it? I just noticed it's gone in the preview
(I never really understood/used it much so I'm fine with removing it)

Edit: It's also soooooo laggy 😂

**Comment by FabianLars** [comment]
didn't even know that there was a search :joy:

**Comment by lorenzolewis** [comment]
@JonasKruckenberg is the live site laggy? or you talking this dev version? If the dev version, then I hand that over to you to help with because #reactland

For the search/filter, figured it wasn't worth it and the current way it was implemented just didn't have a nice behaviour for me imho

**Comment by JonasKruckenberg** [comment]
> @JonasKruckenberg is the live site laggy? or you talking this dev version? If the dev version, then I hand that over to you to help with because #reactland


Sorry I meant the filter functionality of the live site, it's just really laggy. The perf for your release notes themselves is good!

**Comment by lorenzolewis** [comment]
Perfect! @FabianLars just needs your review then I think

**Review by FabianLars — APPROVED** [review]

**Pull request merged**